constructor/header.sh

```diff
@@ -514,21 +518,22 @@ if [ "$PYTHONPATH" != "" ]; then
     printf "    in __NAME__: $PREFIX\\n"
 fi
 
-if [ "$BATCH" = "0" ] && [ "$HAS_CONDA_IN_INSTALL" = "1" ]; then
+if [ "$BATCH" = "0" ]; then
+#if has_conda
     # Interactive mode.
-#if osx
+  #if osx
```

Comment:
I indented these to make it more visually clear that they are nested inside the ``#if has_conda`` block.